www/html/css/index2.css

```diff
@@ -1,20 +1,12 @@
-button.tutortab {
-  background-color: transparent !important;
-  border: transparent !important;
-  letter-spacing: 1px;
-  padding: 0px;
+.navbar-collapse.in {
+    overflow-y: visible !important;
 }
 
 .container.kt-tabs-component {
   margin: 0;
   padding: 0;
 }
 
-li.list-group-item {
```

Comment:
This was made when I was using a collapsible menu in order to it to match the style of the other part. It is no longer needed because I am now reverting back to the original drop down menu